pyproject.toml

```diff
@@ -257,7 +257,6 @@ extend-exclude = [
     "tests/data_asset", # 10 warnings
     "tests/data_context", # 127 warnings
     "tests/dataset", # 10 warnings
-    "tests/datasource", # 81 warnings
```

Comment:
When we moved the experimental tests to `tests/datasource` we lost our linter coverage for tests